ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/monitor-connectivity.yml

```diff
@@ -0,0 +1,31 @@
+---
+monitor_connectivity:
+    shutdown: false
```

Comment:
only two spaces for indentation in yaml files.

Reply:
I will fix this.